apps/search/src/search/models.py

```diff
@@ -388,17 +390,6 @@ def autocomplete(self, autocomplete):
     properties_['autocomplete'] = autocomplete
     self.properties = json.dumps(properties_)
 
-  @property
```

Comment:
We would need this moved to the fixture (https://github.com/cloudera/hue/blob/master/apps/search/src/search/fixtures/initial_search_examples.json) or it would be lost